codebundles/aws-c7n-ec2-health/runbook.robot

```diff
@@ -0,0 +1,105 @@
+*** Settings ***
+Metadata          Author   saurabh3460
+Metadata          Supports    AWS    EC2    CloudCustodian
+Metadata          Display Name    AWS EC2 Health
+Documentation     Check health of EC2 instances to maintain performance and adherence to best practices.
+Force Tags    EC2    Compute    AWS
+
+Library    RW.Core
+Library    RW.CLI
+Library    CloudCustodian.Core
+
+Suite Setup    Suite Initialization
+
+
+*** Tasks ***
+List old AWS EC2 instances in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}` 
```

Comment:
Also, what is the idea behind this task? It seems normal for a user to have a "running" instance older than "60 days"... are the defaults here appropriate? Are we checking the right thing? One task might be to look for EC2 instances that are not in a running state and older than a specific age. Another task might be to look for instances that need a refresh (e.g. running over a certain age, since it implies a lack of patch/update?).

Reply:
Yes, because sometimes instances get patched and restart, and if an instance is quite old, it becomes difficult to patch since it will require numerous updates. This means that old instances can indicate unpatchable systems.

and next code bundle like `AutoScaling Group - Verify ASGs have valid configurations` comes under EC2 instance so I was thinking to include in this EC2 health codebundle

Reply:
and yes agree we can also look for stopped instances as well that too make sense.